installer/README.md

```diff
@@ -18,34 +18,53 @@ Testing on other major cloud providers is in the roadmap.
 
 ## Prerequisites
 
-To use the Cloudflow GKE installer, you need to have the following packages installed on your local machine:
-
+### K8s CLI's
+You need the command line tool for your particular Kubernetes cluster:
 * [Google Cloud SDK](https://cloud.google.com/sdk/)
+* [Amazon CLI for EKS](https://eksctl.io/)
+
+### Utilities
 * [jq](https://stedolan.github.io/jq/)
 * [Helm v2](https://helm.sh/) *note: Cloudflow installer is currently not compatible with Helm v3*
 
 ## Installation Procedure
 
+### GKE
 To install Cloudflow on GKE it is a straightforward process:
 
 ```
 # create a GKE cluster with name <gke-cluster-name>
 $ ./create-cluster-gke.sh <gke-cluster-name>
 # install Cloudflow in that cluster
-$ ./install-gke.sh <gke-cluster-name>
+$ ./install.sh <gke-cluster-name> gke
 ```
 Replace above `<gke-cluster-name>` with the preferred name for
 your GKE cluster.
 
+### EKS
+Similarly, to install Cloudflow on EKS follow this process:
+
+```
+# create an EKS cluster with name <eks-cluster-name>
+$ ./create-cluster-eks.sh <eks-cluster-name> <aws-region>
+# install Cloudflow in that cluster
+$ ./install.sh <eks-cluster-name> eks
+```
+Replace above `<eks-cluster-name>` with the preferred name for
+your EKS cluster.
+
 ## Uninstall Procedure
 
 In case of a failed installation, or if you simply want to uninstall entirely, run `./uninstall-cloudflow.sh`.
 
 Notes
 -----
-- `create-cluster-gke.sh` is optional. 
-It creates a cluster on GKE that's large enough to launch several applications.
-You can also opt to create a cluster customized to your needs by either changing the values in the `create-cluster-gke.sh`, using the [Google Cloud Console](cloud.google.com), or the `gcloud` CLI.
+- `create-cluster-<gke|eks>.sh` is optional. 
+It creates a cluster on GKE/EKS that's large enough to launch several applications.
+You can also opt to create a cluster customized to your needs by either changing the values in the 
+`create-cluster-gke.sh`, using the [Google Cloud Console](cloud.google.com), or the `gcloud` CLI for GKE.
+Similarly, you can also opt to create a cluster customized to your needs by either changing the values in the
+`create-cluster-eks.sh`, using the [Amazon Web Services Console](aws.amazon.com), or the `ekstl` CLI for EKS.
 
 - The `lightbend` namespace
```

Comment:
This has been replaced with `cloudflow` could you please update it as well although could be fixed in another PR.